seed/energy/meter_data_processor/kairos_insert.py

```diff
@@ -70,15 +73,26 @@ def batch_insert_kairosdb(meta_data, ts_data):
         insert_data['tags'] = meta
 
         wrap.append(insert_data)
+        if len(wrap) == KairosDB_Batch_Insert_Size:
+            json_insert_data = json.dumps(wrap)
+            r = requests.post(db_data['insert_url'], data=json_insert_data)
+            if r.status_code != 204:
+                _log.error('Insert Into KairosDB Error ' + str(r.status_code))
+                _log.info('Error Message: ' + r.text)
+                return False
+
+            counter += KairosDB_Batch_Insert_Size
+            _log.info('KairosDB_Inserted, ' + str(counter) + '/' + str(total_len))
+            wrap = []
+
+    if len(wrap) > 0:
+        json_insert_data = json.dumps(wrap)
+        r = requests.post(db_data['insert_url'], data=json_insert_data)
```

Comment:
can we DRY this up a bit? Either make the post a method (and check the response code) or write a chunk method. (http://stackoverflow.com/questions/312443/how-do-you-split-a-list-into-evenly-sized-chunks-in-python)